.github/workflows/ci.yml

```diff
@@ -85,6 +85,52 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
+  #
+  android:
+    runs-on: macos-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    # Definition of the build matrix
+    strategy:
+      matrix:
+        java: [ 11 ]
+        entry:
+          - { mock-maker: 'mock-maker-default', member-accessor: 'member-accessor-default' }
+          - { mock-maker: 'mock-maker-inline', member-accessor: 'member-accessor-module' }
+          - { mock-maker: 'mock-maker-subclass', member-accessor: 'member-accessor-module' }
+          - { mock-maker: 'mock-maker-subclass', member-accessor: 'member-accessor-reflection' }
+          - { mock-maker: 'mock-maker-inline', member-accessor: 'member-accessor-reflection' }
+
+    # All build steps
+    steps:
+    - name: 1. Check out code
+      uses: actions/checkout@v3 # https://github.com/actions/checkout
+      with:
+        fetch-depth: '0' # https://github.com/shipkit/shipkit-changelog#fetch-depth-on-ci
+
+    - name: 2. Set up Java ${{ matrix.java }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'zulu'
+        java-version: ${{ matrix.java }}
+
+    - name: 3. Setup Android SDK
+      uses: android-actions/setup-android@v2
+
+    - name: 4. Install Android SDK components
+      run: |
+        sdkmanager --install "platforms;android-31"
+
+    - name: 5. Run Android tests on Java ${{ matrix.java }} with ${{ matrix.entry.mock-maker }} and ${{ matrix.entry.member-accessor }}
+      uses: reactivecircus/android-emulator-runner@v2
+      with:
+        api-level: 29
+        script: ./gradlew :androidTest:connectedCheck
+      env:
+        MOCK_MAKER: ${{ matrix.entry.mock-maker }}
+        MEMBER_ACCESSOR: ${{ matrix.entry.member-accessor }}
```

Comment:
Could you please add a step (in the same job) for running `gradlew test` too so that unit tests with mocks are also tested. These **should** be just normal JVM tests, but we never know what can break :)

Reply:
ah never mind, that would be covered by normal CI, nice 👏

(iff all the conditions in settings.gradle are met in normal CI)

Reply:
👍